src/NLog.Extensions.Logging/Logging/NLogLoggerProvider.cs

```diff
@@ -64,10 +64,7 @@ public NLogLoggerProvider(NLogProviderOptions options, LogFactory logFactory)
         /// <returns>New Logger</returns>
         public Microsoft.Extensions.Logging.ILogger CreateLogger(string name)
         {
-            var beginScopeParser = ((Options?.CaptureMessageProperties ?? true) && (Options?.IncludeScopes ?? true))
-                ? (_beginScopeParser ?? System.Threading.Interlocked.CompareExchange(ref _beginScopeParser, new NLogBeginScopeParser(Options), null))
-                : null;
```

Comment:
nice that this could be removed :) (lines 67-69). I assume this hasn't a functional impact :)

Reply:
Nope just code cleanup